web-registry/src/lib/db/types/json-ld/cfc-project-metadata.ts

```diff
@@ -9,16 +9,10 @@ export interface CFCProjectMetadataLD extends ProjectMetadataLD {
   'regen:cfcProjectPage': TypeValue;
   'regen:offsetProtocol': OffsetProtocol;
   'regen:projectDesignDocument': TypeValue;
-  'regen:projectOperator'?: ProjectOperator;
```

Comment:
`regen:projectOperator` being removed from CFC, in favor of existing  `regen:projectDeveloper`